specs/DesignDiscussions/InputLabels.md

```diff
@@ -204,59 +169,36 @@ An alternative option would be to have labels as separate text elements, but pro
 	]
 }
 ``` 
-#### Non-adjacent labels
-Like HTML “label for”, this approach doesn’t require the label to be near the input it labels. In most cases, it seems likely that this is not something we want or need to support, and may in fact be confusing to users. However, we should consider if there are valid scenarios for this use case. Possible scenarios that I could come up with include:
 
-##### Labels to the left of the input
+The downside of this is that it provides a less clean story to our card authors. They would now have two ways to specify a label, and without reading the documentation it would not be clear that one option results in host formatting being applied and the other doesn't. If, for example, a card author wants to emphasize a particular word, and uses a RichTextBlock to highlight that word, they might unknowingly opt themselves out of host formatting. 
+
+##### Non-adjacent labels
```

Comment:
This has not been talked about but what happens with Input.Text with inline actions, where does the label go? Only on top of the input or above the input and the action?? the same happens for the error message